src/System.IO.MemoryMappedFiles/src/System/IO/MemoryMappedFiles/MemoryMappedView.Unix.cs

```diff
@@ -18,7 +18,7 @@ internal partial class MemoryMappedView
         {
             if (requestedOffset > memMappedFileHandle._capacity)
             {
-                throw new ArgumentOutOfRangeException("offset");
+                throw new ArgumentOutOfRangeException(nameof(requestedOffset));
```

Comment:
Please revert this.  The name of the parameter in the public entrypoint is "offset".